public/app/plugins/datasource/cloudwatch/components/QueryEditor.tsx

```diff
@@ -1,12 +1,12 @@
 import React, { PureComponent, ChangeEvent } from 'react';
-import { SelectableValue, QueryEditorProps } from '@grafana/data';
+import { SelectableValue, ExploreQueryFieldProps } from '@grafana/data';
 import { Input, Segment, SegmentAsync, ValidationEvents, EventsWithValidation, Switch } from '@grafana/ui';
 import { CloudWatchQuery } from '../types';
 import CloudWatchDatasource from '../datasource';
 import { SelectableStrings } from '../types';
 import { Stats, Dimensions, QueryInlineField, QueryField, Alias } from './';
 
-export type Props = QueryEditorProps<CloudWatchDatasource, CloudWatchQuery>;
+export type Props = ExploreQueryFieldProps<CloudWatchDatasource, CloudWatchQuery>;
```

Comment:
Ok for a quick fix, but really wish we could unify these interfaces, I see very recently that ExploreQueryFieldProps has now a NEW prop in master (compared to 6.5) (absoluteRange?: ) the plan was to get rid of the differences not expand them. Can't find the PR where this was added, do you know?  

history could be handled easily internally by each editor, and onBlur could be added to main interface. not sure what this absoluteRange is used for. PanelData has a timeRange property already.

Reply:
or query history could potentially be made into a more universal feature that we can shared between dashboard / explore.